src/test/java/org/cactoos/http/HtHeadTest.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Random;
 import org.cactoos.io.BytesOf;
 import org.cactoos.io.InputOf;
+import org.cactoos.matchers.TextHasString;
```

Comment:
@Vatavuk please use `org.llorllale.cactoos.matchers.TextHasString` (we'll soon be removing `org.cactoos.matchers` from the classpath)